src/Apache-1.1.xml

```diff
@@ -43,18 +43,14 @@
           <alt match="name\(s\)|name|name" name="nameClause4">name</alt>
             <alt match=".+" name="organizationClause4">"Apache" and "Apache Software Foundation"</alt> must
           not be used to endorse or promote products derived from this software without prior written
-          permission. For written permission, please contact
-          <alt match=".+" name="contactClause4">apache@apache.org</alt>.
```

Comment:
Why was the markup around the email removed?

Reply:
@agr The PR was done a while go, so I don't recall the reason.  Likely changing the entire sentence to optional seemed to obviate the need for the alternative clause around the contact.

There is no harm, however, in including the alternative tag as long as the optional tag is retained.

If removing this is causing an issue, feel free to create a PR adding it back.  If it causes an issue in the tests, let me know.